extensions/cornerstone-dicom-sr/src/index.tsx

```diff
@@ -30,7 +32,34 @@ const dicomSRExtension = {
    * Only required property. Should be a unique value across all extensions.
    */
   id,
-  onModeEnter,
+
+  onModeEnter({ servicesManager }) {
+    const { toolbarService } = servicesManager.services;
+
+    toolbarService.addButtons([
+      {
+        // A default button for loading measurements added to the toolbar below.
+        // Customizations to this button can be made in the mode using this extension.
+        // For example, the button label can be changed and/or command props to
+        // not clear the measurements can be passed to the command.
+        id: 'LoadMeasurements',
+        component: ViewportActionButton,
+        props: {
+          label: i18n.t('Common:LOAD'),
+          commands: [
+            {
+              commandName: 'loadMeasurements',
+              context: 'CORNERSTONE_STRUCTURED_REPORT',
```

Comment:
i think we don't need context anymore, we removed that, ask Bill please

Reply:
@wayfarer3130, comments?

Reply:
No, the context isn't required, and leaving it out means one can over-ride the command in the tracked measurements extension if we decide to replace it.